api/src/org/apache/cloudstack/api/command/admin/host/PrepareForMaintenanceCmd.java

```diff
@@ -108,4 +108,8 @@ public void execute() {
             throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, "Failed to prepare host for maintenance");
         }
     }
+
+    public void setHostId(final Long hostId) {
+        id = hostId;
```

Comment:
this setter is still to be called setId(..) instead of setHostId(..), or should the field be renamed?

Reply:
Can fix that